media/js/firefox/whatsnew/experiment-whatsnew-70.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+(function() {
+    'use strict';
+
+    // courtesy of https://davidwalsh.name/query-string-javascript
+    function getUrlParam(name) {
+        name = name.replace(/[[]/, '\\[').replace(/[\]]/, '\\]');
+        var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
+        var results = regex.exec(location.search);
+        return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
+    }
+
+    // if we already have a `v` parameter on the page don't enter into the experiment.
+    if (!getUrlParam('v')) {
+        var wallander = new Mozilla.TrafficCop({
+            id: 'experiment_firefox_whatsnew_70',
+            variations: {
+                'v=0': 65, // control
```

Comment:
The experiment plan says `Each group (including the control): should get 5% traffic distribution`, so I don't think this should be at 65%?

Reply:
You're right. I realized far too late that I had set it up for 100% enrollment. With the new `entrypoint_variation` param I think I need to set up a proper v=0 template just to pass that.